File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Linq;

public class Solution
{
    public int solution(string[] babbling)
    {
        int answer = 0;

        string[] baby = { "aya", "ye", "woo", "ma" };

        for (int i = 0; i < babbling.Length; i++)
        {
            for (int j = 0; j < baby.Length; j++)
            {
                babbling[i] = babbling[i].Replace(baby[j], "1");
            }

            if (babbling[i].Trim('1') == "")
            {
                answer++;
            }
        }

        return answer;
    }
}

File: 프로그래머스/unrated/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public static int solution(String[] babbling) {
        int answer = 0;

        String[] aya = {"aya", "ye", "woo", "ma"};

        for (int i = 0; i < babbling.length; i++) {

            String s = babbling[i];

            for (int j = 0; j < aya.length; j++) {

                if (babbling[i].contains(aya[j])) {
                    s = s.replace(aya[j], " ");
                }
            }

            babbling[i] = s;

            if (babbling[i] != null) {
                String s2 = babbling[i].trim();
                if (s2.equals("")) {
                    answer++;
                }
            }
        }

        return answer;
    }
}